massa-time/src/lib.rs

```diff
@@ -415,7 +370,7 @@ impl MassaTime {
     /// assert_eq!(massa_time.format_instant(), String::from("2022-01-01T00:00:00Z"))
     /// ```
     pub fn format_instant(&self) -> String {
```

Comment:
are there cases where we would print / format unchecked inputs ? if so, this could be a source of panic attacks

Reply:
Yes maybe it could be it's not checked anywhere but it wasn't on the security team input issue will look into it after.

Reply:
ok, can we open a followup issue on this ?

Reply:
I have made the change just now. Tell me if it's good for you :) !

Reply:
sounds good :)